dart/examples/web_example/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: sentry_dart_web_example
+description: An absolute bare-bones web app.
```

Comment:
I wonder what happens to pub.dev when there are folders in the example directory.

Today it just shows one file: https://pub.dev/packages/sentry/example

Reply:
good question

Reply:
I move back the io.example to `/example` and added a `/example_web` folder